docker/snippets/agx.dockerfile

```diff
@@ -0,0 +1,9 @@
+# syntax = devthefuture/dockerfile-x
+# SPDX-License-Identifier: MIT
+# This snippet helps with setup on agx specific systems
+
+# Add Isaac apt repositories
+# Without this, humble packages on focal (the default ubuntu version for AGX) can't be found
+RUN wget -qO - https://isaac.download.nvidia.com/isaac-ros/repos.key | apt-key add - && \
```

Comment:
saw this, but how does this help with our issues (pytorch)?

Reply:
This will correctly setup `apt` to search for humble packages, which won't be the case with the regular ros apt keys cause we're using focal, which doesn't have apt binaries for humble. I guess Isaac set it up so that on focal, you can still find humble binaries.